docs/release-notes.md

```diff
@@ -1,3 +1,9 @@
+## 3.7
+
+<h5>Infrastructure Changes</h5>
+
+Zanata now uses Infinispan as its cache provider, and the cache needs to be configured in Jboss' `standalone.xml` file. Please see the [Infinispan](configuration/infinispan) section for more information.
+
```

Comment:
I don't think this pull request updates all SIX! copies of standalone*.xml in the source tree.